ansible/playbooks/roles/upgrade/defaults/main.yml

```diff
@@ -5,7 +5,7 @@ logging:
       cert_path: /etc/elasticsearch/custom-admin.pem
       key_path:  /etc/elasticsearch/custom-admin-key.pem
 
-opendistro_for_elasticsearch:
+opensearch:
```

Comment:
It would be best to do the new approach for the upgrade mode.
I suggest moving all of the upgrade tasks for opensearch to:
roles/opensearch/upgrade/

Reply:
Maks sense to me, however due to the large scope of the current change I would suggest to reference it a new issue, as this seems to be rather a code optimization change than a deal breaker.